addons/udes_sale_stock/tests/test_stock_picking.py

```diff
@@ -0,0 +1,66 @@
+from . import common
+
+
+class TestStockPicking(common.BaseSaleUDES):
+    @classmethod
+    def setUpClass(cls):
+        super(TestStockPicking, cls).setUpClass()
+
+        Customer = cls.env["res.partner"]
+
+        cls.customer = Customer.create(
```

Comment:
Do we want to do this at `common.BaseSaleUDES`? The same customer is created in three different test files.

Reply:
Moved creation to common as suggested